src/lib.rs

```diff
@@ -365,8 +365,9 @@ impl Client {
     }
 
     /// Delete multiple keys
-    pub async fn delete_multi_no_reply<K>(&mut self, keys: &[K]) -> Result<(), Error>
+    pub async fn delete_multi_no_reply<I, K>(&mut self, keys: I) -> Result<(), Error>
     where
+        I: IntoIterator<Item = K>,
```

Comment:
I think it's actually more idiomatic to omit the `.into_iter()` in this case since it is already when using something that implements `IntoIterator` in a for loop: (source)[https://doc.rust-lang.org/std/iter/#for-loops-and-intoiterator].